crates/l2/contracts/deployer.rs

```diff
@@ -755,8 +755,26 @@ async fn make_deposits(bridge: Address, eth_client: &EthClient) -> Result<(), De
                 DeployError::DecodingError("Error while parsing private key".to_string())
             })?;
         let address = get_address_from_secret_key(&secret_key)?;
-        let values = vec![Value::Address(address)];
-        let calldata = encode_calldata("deposit(address)", &values)?;
+        let values = vec![
+            Value::Address(address),
+            Value::Address(address),
+            Value::Uint(U256::from(21000 * 5)),
+            Value::Bytes(Bytes::from_static(b"")),
+        ];
+
+        // This should be changed once https://github.com/lambdaclass/ethrex/issues/2384 is addressed
```

Comment:
I'd add a `FIXME: ` prefix to the comment so it's clearer that it should be changed